docs/frontend/setup/designers-guide-to-setting-up-app-locally.md

```diff
@@ -66,15 +66,45 @@ For the sake of time let's try to install all of these dependencies at once. Som
 
 Environment variables define things such as database credentials, security certificates, API keys, and feature flags that can vary between environment (local, experimental, staging, production).
 
-Most developers have AWS credentials that are able to keep these values synced and up to date.  In the scenario you don't have AWS permissions we will need to create a file with these pre-defined values filled in.  This file could become outdated as values change, get added or removed.  We should take care not to share this file publicly eventhough most of the values are not the same secrets we use elsewhere.
+Most developers have AWS credentials that are able to keep these values synced and up to date. In the scenario you don't have AWS permissions we will need to create a file with these pre-defined values filled in. This file could become outdated as values change, get added or removed. We should take care not to share this file publicly eventhough most of the values are not the same secrets we use elsewhere.
 
 Ask another developer with AWS credentials to send you the output of running:
 
-`DISABLE_AWS_VAULT_WRAPPER=1 AWS_REGION=us-gov-west-1 aws-vault exec transcom-gov-dev -- chamber list app-devlocal`
+```
+DISABLE_AWS_VAULT_WRAPPER=1 AWS_REGION=us-gov-west-1 aws-vault exec transcom-gov-dev -- chamber list app-devlocal
```

Comment:
I appreciate this call out, but I think for the OSS docs we want to point to Confluence documenting Chamber as it's more reliable and up to date with the system changes at the InfraSec level.

Reply:
Removed and added link to chambers page